library/VTKExtensions/Rendering/Testing/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-# TODO add actual tests
+if(VTK_VERSION VERSION_LESS_EQUAL 9.1.0)
+  cmake_policy(SET CMP0115 OLD)
```

Comment:
Why is it needed?

Reply:
VTK must be doing something strange internally with the file extensions. Without this policy with VTK 9.0.X, there is a CMake warning.